docs/reference/release-notes/highlights.asciidoc

```diff
@@ -23,7 +23,27 @@ Other versions:
 | {ref-bare}/7.1/release-highlights-7.1.0.html[7.1]
 | {ref-bare}/7.0/release-highlights-7.0.0.html[7.0]
 
+// tag::notable-highlights[]
+[discrete]
+[[data-tiers]]
+=== Data Tier Formalization
+7.10 introduces the concept of formalized data tiers within Elasticsearch. <<data-tiers,Data tiers>>
+are a simple, integrated approach that hopes to give users control over optimizing for cost,
+performance, and breadth/depth of data. Prior to this formalization, many users configured their own
+tier topology using custom node attributes as well as using {ilm-init} to manage the lifecycle and
+location of data within a cluster.
+
+With this formalization, data tiers (content, hot, warm, and cold) can be explicitly configured
+using <<node-roles,node roles>>, and indices can be configured to be allocated within a specific
+tier using <<data-tier-shard-filtering,index-level data tier allocation filtering>>. {ilm-init} will
+make use of these tiers to <<automatically ilm-migrate,migrate>> data between nodes as an index goes
+through the phases of its lifecycle.
 
+Newly created indices *not* part of a <<data-stream,data stream>> will be allocated to
+`data_content` tier automatically, while indices abstracted by a data stream will be allocated to
+the `data_hot` tier automatically. Nodes with the pre-existing `data` role are considered to be part
+of all tiers.
```

Comment:
I'd flip this paragraph around to avoid starting with a negation. Maybe just a personal pereferent but the immediate "*not*" bears a bit more cognitive load.

```suggestion
Newly created indices abstracted by a <<data-stream,data stream>> will be allocated to
the `data_hot` tier automatically, while standalone indices will be allocated to
the `data_content` tier automatically. Nodes with the pre-existing `data` role are 
considered to be part of all tiers.
```